presto-client/src/main/java/com/facebook/presto/client/QueryResults.java

```diff
@@ -312,6 +314,8 @@ private static Object fixValue(TypeSignature signature, Object value)
             case INTERVAL_DAY_TO_SECOND:
             case DECIMAL:
                 return String.class.cast(value);
+            case CHAR:
+                return Strings.padEnd(String.class.cast(value), getCharTypeLength(signature), ' ');
```

Comment:
Padding on the client side doesn't seem like the right thing to do to me, especially since padding is a configurable option in the server